src/lib/y2partitioner/actions/controllers/btrfs_devices.rb

```diff
@@ -225,8 +222,6 @@ def create_filesystem(device)
           filesystem.metadata_raid_level = @metadata_raid_level
           filesystem.data_raid_level = @data_raid_level
 
-          UIState.instance.select_row(filesystem.sid)
```

Comment:
How is the row of the new filesystem selected now? For example, when we go to the list of Btrfs and we add a new one.

Reply:
Because the latest steps of the process are not performed on this controller, but by the `Filesystem` one. See
https://github.com/yast/yast-storage-ng/blob/c28f15ad9cec7a4e011ad9de6c39e8e08efb1ac3/src/lib/y2partitioner/actions/add_btrfs.rb#L51